doc/source/api/solver/reduction.rst

```diff
@@ -42,6 +42,11 @@ an area-average of absolute pressure over the velocity inlet.
 
 Similarly one can use the other functions available currently with PyFluent.
 
+.. note::
+   For reduction functions like "volume", "mass_flow", "mass_average" and,
```

Comment:
- should be no comma after and
- we are supposed to put a comma before and even though there's no good reason for it